config/auth.php

```diff
@@ -112,4 +112,5 @@
         ],
     ],
 
+    'iar' => env('APP_REGISTER', false),
```

Comment:
Do we need to cast to `bool` here to ensure the ENV setting (a string) is correctly evaluated down to a type-safe boolean in code?